src/analysis/interpolation/Bezier3D.h

```diff
@@ -20,45 +20,76 @@
 #include "ParametricLine.h"
 #include "qgslogger.h"
 
-/** \ingroup analysis
+/**
+ * \ingroup analysis
  * Class Bezier3D represents a bezier curve, represented by control points. Parameter t is running from 0 to 1. The class is capable to calculate the curve point and the first two derivatives belonging to t.*/
 class ANALYSIS_EXPORT Bezier3D: public ParametricLine
 {
   protected:
 
   public:
-    /** Default constructor*/
+
+    /**
+     * Default constructor*/
```

Comment:
Maybe we should auto-flip these to a single line `//! ` tag?

I realised the trailing */ issue too. That definitely needs addressing.